http/headers/Content-Security-Policy.json

```diff
@@ -1004,7 +1004,7 @@
                 "opera": "mirror",
                 "opera_android": "mirror",
                 "safari": {
-                  "version_added": null
+                  "version_added": "15.6"
```

Comment:
Tested this using https://wpt.live/content-security-policy/script-src/script-src-sri_hash.sub.html . v 15.6 is the earliest version I have to check with that works on Browserstack. It could in theory I be earlier.